app/views/publishers/change_email.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for(:head_style_tags) do
+  = stylesheet_link_tag("application", media: "all")
+
+.single-panel--wrapper
+  = render "panel_flash_messages"
+  .single-panel--content
+
+    h3.single-panel--headline= t ".login_email"
+
+    .col-small-centered
+      p
+        span= t ".login_email_message"
+
+      = form_for @publisher, url: update_email_publishers_path do |f|
```

Comment:
are you able to run this? i tried to change my email and nothing happened.

i think because there's no HTML handler within `Publishers #update`.

we should test that change_email is able to actually change emali